src/lib/y2partitioner/icons.rb

```diff
@@ -42,5 +46,8 @@ module Icons
 
     # icon
     ENCRYPTED = "yast-encrypted.png".freeze
+
+    # path to small icons, fits nicely in table
+    SMALL_ICONS_PATH = (Yast::Directory.icondir + "22x22/apps/").freeze
   end
```

Comment:
What about defining a helper for building an icon path?

```ruby
def small_icon(icon)
  SMALL_ICONS_PATH + icon
end
```

Then replace `icon = Icons::SMALL_ICONS_PATH + Icons::HD` by `icon = Icons::small_icon(Icons::HD)`.

And even maybe define generic `icon_path` helper:

```ruby
def icon_path(icon)
  small_icon(icon)
end
```

Then you can easily change the default icon size globally at a single place:
```ruby
def icon_path(icon)
  big_icon(icon)
end
```